app/controllers/api/v2/advisor_engine/advisor_engine_controller.rb

```diff
@@ -17,6 +22,72 @@ def host_details
             end
           end
         end
+
+        api :PATCH, "/advisor_engine/upload_hits", N_("Upload hits from foreman-advisor-engine")
```

Comment:
I got the name wrong before. :melting_face: 

```suggestion
        api :PATCH, "/advisor_engine/upload_hits", N_("Upload hits from iop-advisor-engine")
```